libraries/lib-time-and-pitch/tests/StaffPadTimeAndPitchTest.cpp

```diff
@@ -0,0 +1,116 @@
+#include "StaffPadTimeAndPitch.h"
+#include "AudioContainer.h"
+#include "WavFileIO.h"
+
+#include <catch2/catch.hpp>
+
+namespace fs = std::filesystem;
+using namespace std::literals::string_literals;
+
+namespace
+{
+TimeAndPitchInterface::AudioSource
+MakeAudioSource(const std::vector<std::vector<float>>& input)
+{
+   auto numPulledFrames = std::make_shared<unsigned long long>();
+   *numPulledFrames = 0;
+   return [&input,
+           numPulledFrames](float* const* buffer, size_t samplesPerChannel) {
```

Comment:
Given that AudioSource is never copied in StaffPadTimeAndPitch, I think that `numPulledFrames = 0ULL` is sufficient here